doc/examples/denoise_ascm.py

```diff
@@ -123,18 +123,17 @@
 
 print("The ascm result saved in denoised_ascm.png")
 
-"""
-.. figure:: denoised_ascm.png
-   :align: center
-
-   Showing the axial slice without (left) and with (middle) ASCM denoising.
-"""
 
-"""
-From the above figure we can see that the residual is really uniform in nature
-which dictates that ASCM denoises the data while preserving the sharpness of
-the features.
-"""
+############################################################################
+#
+# .. figure:: denoised_ascm.png
```

Comment:
I do not think this is the correct name, I suppose it start with `sphx_glr_denoised_ascm.png`, I need to check and test it